server/review.test.js

```diff
@@ -0,0 +1,51 @@
+const request = require('supertest')
+  , {expect} = require('chai')
+  , db = require('APP/db')
+  , app = require('./start')
+
+/* global describe it before afterEach */
+
```

Comment:
Please add the following comment!

// To do: These test need to be re-written at a later point.